source/API_Reference/Event_Webhook/event.md

```diff
@@ -0,0 +1,1043 @@
+{% endwarning %}
+
+{% anchor h4 %}
+Reserved Keys in Unique Arguments
+{% endanchor %}
+
+{% codeblock lang:json %}
+//for this example, assume we're sending to john.doe@sendgrid.com
+{
+  "unique_args": {
+    "customerAccountNumber": "55555",
+    "activationAttempt": "1",
+    "New Argument 1": "New Value 1",
+    "email": "jane.doe@sendgrid.com",
+    "event": "SendEmail"
+  }
+}
+{% endcodeblock %}
+
+{% anchor h4 %}
+The resulting webhook call
+{% endanchor %}
+
+{% codeblock lang:json %}
+[
+  {
+    "event": "Processed",
+    "timestamp":"123456789",
+    "customerAccountNumber": "55555",
+    "activationAttempt": "1",
+    "New Argument 1": "New Value 1",
+    "email": "john.doe@sendgrid.com"
+  }
+]
+{% endcodeblock %}
+
+*Note: The event and the email do not get overwritten because "event" and "email" are reserved, and SendGrid defaults to the reserved key for webhooks.*
+
+{% anchor h3 %}
+Custom Arguments
+{% endanchor %}
+
+Any custom arguments that you include with an email sent through [v3 Mail Send]({{root_url}}/API_Reference/Web_API_v3/Mail/index.html) gets added to your Event Webhook response.
+
+For example, if you were to include the following custom arguments in a personalization in your payload to the v3 Mail Send endpoint:
+
+{% codeblock lang:json %}
+{
+  "personalizations": [
+    {
+      "to": [
+        {
+          "email": "example@example.com"
+        }
+      ],
+      "subject": "Hello, World!",
+      "custom_args": {
+        "userid": "1123"
+      }
+    }
+  ],
+  "from": {
+    "email": "from_address@example.com"
+  },
+  "content": [
+    {
+      "type": "text/plain",
+      "value": "Hello, World!"
+    }
+  ]
+}
+{% endcodeblock %}
+
+The Event Webhook response:
+
+{% codeblock lang:json %}
+[
+  {
+    "userid": "1123"
+  }
+]
+{% endcodeblock %}
+
+- <a name="marketingcampaignid"></a>`marketing_campaign_id`
+- <a name="marketingcampaignname"></a>`marketing_campaign_name`
+
+For emails sent through our Marketing Campaigns feature, we will add Marketing Campaigns specific params to the Event Webhook. Both `marketing_campaign_name` and `marketing_campaign_id` are displayed as unique arguments in the event data.
+
+{% anchor h3 %}
+Example event from a standard (non-A/B test) campaign send:
+  {% endanchor %}
+
+{% codeblock lang:json %}
+  {
+    "category": [],
+    "email": "email@example.com",
+    "event": "processed",
+    "marketing_campaign_id": 12345,
+    "marketing_campaign_name": "campaign name",
+    "post_type": "event",
+    "sg_event_id": "sendgrid_internal_event_id",
+    "sg_message_id": "sendgrid_internal_message_id",
+    "sg_user_id": 12345,
+    "smtp-id": "",
+    "timestamp": 1442349428
+  }
+{% endcodeblock %}
+
+{% anchor h3 %}
+Example event from an A/B Test:
+{% endanchor %}
+
+ `marketing_campaign_version` will be displayed in the event data for emails sent as part of an A/B Test. The value for `marketing_campaign_version` are returned as `A`, `B`, `C`, etc.
```

Comment:
+ `marketing_campaign_version` ~~will be~~ is displayed in the event data for emails sent as part of an A/B Test. The value for `marketing_campaign_version` are returned as `A`, `B`, `C`, etc.